wurst/systems/crafting/CraftingArmory.wurst

```diff
@@ -0,0 +1,66 @@
+package CraftingArmory
+
+// Extension library imports:
+import ItemRestriction
+
+// Local imports:
+import CraftingExtensions
+import ItemRecipeExtended
+import LocalObjectIDs
+import GameMode
+
+
+init
+    GameMode.onModeSelectionFinish() ->
+        let armory = new UnitRequirement("armory")
+        armory.addUnit(UNIT_ARMORY)
+
+        let anabolicBootsWithCharges = new ItemRecipeExt(ITEM_ANABOLIC_BOOTS, 0, true, true, true, ABILITY_QM_ANABOLIC_BOOTS)
+        ..addBoneBoots()
+        ..addSpiritWind()
+        ..addSpiritWater()
+        ..addManaCrystal(true)
+        // Note : Now you will have to stack the potions to craft anabolic boots
```

Comment:
I don't think you need to note this in code. In general comments shouldn't be a changelog because comments are permanent. Feel free to add that with `$changelog` for this PR though.